Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -214,9 +214,17 @@
         m_delegateMethods.webViewRequestPermissionForXRSessionOriginModeAndFeaturesWithCompletionHandler = true;
     else
         m_delegateMethods.webViewRequestPermissionForXRSessionOriginModeAndFeaturesWithCompletionHandler = [delegate respondsToSelector:@selector(_webView:requestPermissionForXRSessionOrigin:mode:grantedFeatures:consentRequiredFeatures:consentOptionalFeatures:completionHandler:)];
-    m_delegateMethods.webViewStartXRSessionWithCompletionHandler = [delegate respondsToSelector:@selector(_webView:startXRSessionWithCompletionHandler:)];
-    m_delegateMethods.webViewEndXRSession = [delegate respondsToSelector:@selector(_webViewEndXRSession:)];
+
+#if TARGET_OS_IPHONE
```

Comment:
There's a comment in `wtf/Platform.h` saying:
```

/* FIXME: Any remaining use of TARGET_OS_IPHONE should be removed outside of Apple only files and replaced with OS() checks. */
```
so I'm not sure if this should be replaced by something else here.

Reply:
Thanks! Addressed here: https://bugs.webkit.org/show_bug.cgi?id=264857